client/src/splitViewHandle.js

```diff
@@ -0,0 +1,177 @@
+import { LuigiClientBase } from './baseClass';
+import { helpers } from './helpers';
+
+/**
+ * The Split View 
+  - Actions to control the split view
+  - Event handlers
+  - Get the current state
+  * @name splitView
+  */
+export class splitViewHandle extends LuigiClientBase {
+  /**
+   * @private
+   */
+  constructor(settings) {
+    super();
+
+    this.splitView = {
+      exists: true,
+      size: 0,
+      isCollapsed: false,
+      isExpanded: true
```

Comment:
Ad discussed, if this object is only for internal use, maybe we don't need `isExpanded`, and as `isCollapsed` is not a function we could call it just `collapsed`

Reply:
done